features/step_definitions/functional_tests/functional_tests_steps.rb

```diff
@@ -30,10 +30,17 @@
 Then(/^all the functional tests for the package "([^"]*)" pass$/) do |package_name|
   tests_dir = "/var/lib/#{package_name}-test"
   package_core_name = package_name.match(/python-(.+)/).captures.first
+  ssl_insecure =
+    case package_name
+    when "python-novaclient"
+      json_response = JSON.parse(admin_node.exec!("crowbar nova show default").output)
```

Comment:
We have added the `proposal` helper recently that handles both crowbar request and json data: https://github.com/SUSE-Cloud/cct/blob/master/features/support/feature_helpers.rb#L4